server/protocols/webadmin-integration-test/webadmin-integration-test-common/src/test/java/org/apache/james/webadmin/integration/EventDeadLettersIntegrationTest.java

```diff
@@ -61,15 +59,13 @@
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Ignore;
-import org.junit.Rule;
 import org.junit.Test;
-
-import com.google.inject.multibindings.Multibinder;
+import org.junit.experimental.categories.Category;
 
 import io.restassured.RestAssured;
 import io.restassured.http.ContentType;
 
-public class EventDeadLettersIntegrationTest {
+public abstract class EventDeadLettersIntegrationTest {
```

Comment:
Please annotate this class with @category(BasicFeature.class)

Also, we don't need an abstract class as only distributed is running this test.